specs/DesignDiscussions/InputLabels.md

```diff
@@ -72,17 +72,55 @@ This conflict speaks to a general tension inherent in the design of Adaptive Car
 
 In order to gather more data to help us balance the needs of the card authors and hosts, we will send a survey to customers who have expressed an interest in improving out input story. The following options are under consideration, and will be evaluated against survey data to determine which if any will be included in our v1 implementation.
 
+> ## Rendering for string labels
+>
+> The default rendering for labels should be performed with the default values for all of the following properties:
+> * spacing
+> * fontFamily
+> * fontSizes
+> * fontWeights
+> * foregroundColors
+>
+> This would allow authors to emulate writing a simple TextBlock element which only non-default property would be the text, making the string version look like this
+>
+> ```json
+> {
+>	"type": "Input.Text",
+>	"id": "textInput",
+>	"label": "Label for Input"
+> }
+> ```
+>
+> While the TextBlock version would look like this
+>
+> ```json
+> {
+>	"type": "Input.Text",
+>	"id": "textInput",
+>	"label": {
+>		"type": "TextBlock",
+>		"text": "Label for Input"
+>	}
+> }
+> ```
+>
+> Both of the afore mentioned cards would produce the same rendered result that would look like  this:
+> 
+> ![img](assets/InputLabels/LabelDefault.PNG)
+>
+
 ### Host Formatting
 
 If we want to allow the host to control formatting of the labels, either in general or for required inputs, we need a way for them to specify the desired format. There are two main things we may want to include:
 
 #### Text formatting
  We can introduce the option to specify formatting of input labels for required and optional inputs. This could include things such as bold, italics, size, etc. We could either allow full text formatting (as in the formatting properties on a text block) or a subset specified as an enum. Since this would be host control, full formatting is probably fine here. For example:
```

Comment:
>llow full text formatting [](start = 178, length = 25)

Are there any blockers to supporting fulltext formatting at this point? If not, lets just then go with that approach in the proposal here to close unless there are open comments/issues/implementation costs here that are making this complicated.